internal/server/email/templates/signup.text.html

```diff
@@ -2,5 +2,9 @@ <h3>Welcome to Infra!</h3>
 
 <p>
   You can sign into your account any time from 
-  <a href="{{.Link}}">{{.Link}}</a>.
+  <a href="{{.WrappedLink}}">https://{{.Link}}</a>.
```

Comment:
I believe the link already has the scheme on it, from what I can see in `API.Signup`
```suggestion
  <a href="{{.WrappedLink}}">{{.Link}}</a>.
```

It'd be great to have some test coverage using `gotest.tools/v3/golden` to be able to preview the fully rendered email. Not a blocker for this PR though.

Reply:
we can already preview rendered emails in tests. see TestSendEmail

Reply:
added an example of this